lib/tasks/sandbox_data.rake

```diff
@@ -15,4 +15,16 @@ namespace :lead_providers do
     end
     ValidTestDataGenerator::AmbitionSpecificPopulater.call(name: "Ambition Institute", total_schools: 3, participants_per_school: 1500)
   end
+
+  desc "create NPQ seed schools and participants for API testing"
+  task seed_schools_and_participants_npq: :environment do
+    logger = Logger.new($stdout)
+    logger.formatter = proc do |_severity, _datetime, _progname, msg|
+      "#{msg}\n"
+    end
+
+    ["Ambition Institute", "Capita", "Teach First", "UCL Institute of Education", "Best Practice Network", "Education Development Trust"].each do |provider|
+      ValidTestDataGenerator::NPQLeadProviderPopulater.call(name: provider, total_schools: 100, participants_per_school: 100)
```

Comment:
Again. These should match up with the generated NPQ lead providers rather than the ECF ones.